src/mongocxx/test/CMakeLists.txt

```
@@ -206,23 +206,27 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 target_link_libraries(test_driver PRIVATE Threads::Threads)
 
-add_test(NAME driver COMMAND test_driver)
-add_test(NAME logging COMMAND test_logging)
-add_test(NAME instance COMMAND test_instance)
-add_test(NAME crud_specs COMMAND test_crud_specs)
-add_test(NAME gridfs_specs COMMAND test_gridfs_specs)
-add_test(NAME client_side_encryption_specs COMMAND test_client_side_encryption_specs)
-add_test(NAME command_monitoring_specs COMMAND test_command_monitoring_specs)
-add_test(NAME transactions_specs COMMAND test_transactions_specs)
-add_test(NAME retryable_reads_spec COMMAND test_retryable_reads_specs)
-add_test(NAME read_write_concern_specs COMMAND test_read_write_concern_specs)
-add_test(NAME unified_format_spec COMMAND test_unified_format_spec)
-add_test(NAME versioned_api COMMAND test_versioned_api)
+foreach(test_name
+    client_side_encryption_specs
+    command_monitoring_specs
+    crud_specs
+    driver
+    gridfs_specs
+    instance
+    logging
+    read_write_concern_specs
+    retryable_reads_spec
```

```suggestion
    retryable_reads_specs
```

To fix tasks [failing with](https://spruce.mongodb.com/task/mongo_cxx_driver_integration_versioned_api_vs2019_latest_single_test_versioned_api_accept_version_two_patch_54693de778fffb56571853d7f22c7e972d53b845_66e081132288530007969da0_24_09_10_17_25_40/logs?execution=0) "Unable to find executable: test_retryable_reads_spec"